test/synapse/2019-06-01/azspark/zz_generated_constants.go

```diff
@@ -9,8 +9,8 @@
 package azspark
 
 const (
-	telemetryInfo = "azsdk-go-azspark/" + version
-	version       = "v0.1"
+	module  = "azspark"
+	version = "v0.1"
```

Comment:
We should standardize our versions v0.1.0 keeping the Major.Minor.Rev

Reply:
fixed